docs/README.md

```diff
@@ -2,8 +2,9 @@
   <img style='width: 500px' src="assets/mvi_logo.png" alt="adidas mvi Logo"/>
 </div>
 
+[![Kotlin](https://img.shields.io/badge/Kotlin-1.8.20-blue.svg?style=flat&logo=kotlin)](https://kotlinlang.org)
```

Comment:
would be nice if we can bump this when a new release become available.

Reply:
this is the one we use on the project, we need to bump it when also updating the Kotlin version of the project.